olp-cpp-sdk-dataservice-read/src/repositories/ApiCacheRepository.cpp

```diff
@@ -41,15 +44,18 @@ void ApiCacheRepository::Put(const std::string& service,
                              const std::string& serviceVersion,
                              const std::string& serviceUrl) {
   std::string hrn(hrn_.ToCatalogHRNString());
+  auto key = CreateKey(hrn, service, serviceVersion);
+  LOG_TRACE_F(kLogtag, "Put '%s'", key.c_str());
```

Comment:
Why trace and not info?
Same below.

Reply:
as per decision to put methods entry at trace level

Reply:
Methods entry yes but you are printing parameters.
Method entry trace log should be without parameters print:
E.g.
```
LOG_TRACE_F(kLogTag, "%s", __PRETTY_FUNCTION__);
```